subprojects/gradle/ci-logger/src/test/kotlin/com/avito/utils/logging/CILoggingDestinationTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.avito.utils.logging
+
+import com.avito.android.sentry.SentryConfig
+import org.junit.jupiter.api.Test
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.io.ObjectOutputStream
+
+class CILoggingDestinationTest {
+
+    @Test
+    fun `destinations are serializable`() {
+        assertSerializable(StdoutDestination)
+        assertSerializable(StderrDestination)
+        assertSerializable(OnlyMessageStdoutDestination)
+        assertSerializable(
+            SentryDestination(
+                config = SentryConfig.Enabled(
+                    dsn = "stub",
+                    environment = "stub",
+                    serverName = "stub",
+                    release = "stub",
+                    tags = emptyMap()
+                )
+            )
+        )
+        assertSerializable(
+            FileDestination(
+                file = File(".")
+            )
+        )
+        // elastic is not supported yet
```

Comment:
it is supported now, right?

Reply:
No, https://github.com/avito-tech/avito-android/pull/678/files#r544201765

Reply:
Fixed